Makefile

```diff
@@ -12,6 +12,12 @@ include $(config)
 export LDFLAGS = -pthread -lm
 export CFLAGS = -std=c++11 -Wall -O2 -Iinclude -fPIC
 
+ifdef DMLC_CORE_PATH
```

Comment:
hmm, after double checked the path configuration in mxnet. I think the ifdef way is actually better. So let us go with the current approach